specs/0111-redesign-profile-page.md

```diff
@@ -0,0 +1,113 @@
+- Feature Name: Redesign of the Profile Page
+- Created: 2022-03-30
+- Associated Issue: #2028, #2203, #2112
+
+# Summary
+
+Redesign the profile page to better serve both users and administrators by making data about contributions and contact information easy to access.
+
+# Motivation
+
+The current profile page only provides minimal information on a contributor. The timeline is not interactive and shows only some of the translations, forcing someone interested in knowing more about specific contributions to manually search for them in Pontoon.
+
+Contributors should be able to find all the information about their work in the profile page, and use it to promote themselves internally, e.g. to ask locale managers or administrators for translator permissions, and externally, e.g. to prove their experience as a volunteer. Their contributions should be easily accessible through links in the profile page.
+
+Locale managers and administrator should be able to determine the quality of contributions from the profile page, and find all the relevant contact information to get in touch.
+
+# Feature explanation
+
+See the [mockup](#mockup) for the overall view. Each of the following sections provides more details on different parts of the design.
+
+## Latest activity
+
+*Latest activity* is displayed in the sidebar. A user is considered active when they either submit a suggestion or review someone else’s work. It’s not considered activity:
+* Logging in to Pontoon.
+* Writing a comment.
+
+## Approval ratio
+
+Displayed in a graph, it’s the ratio between the number of translations approved over the total number of translations submitted, excluding self-approved translations (either submitted directly as translations, or provided as suggestions and later approved by the translator).
+
+Examples:
+* If a person submits 100 translations, but 20 are rejected, the approval ratio will be 80%.
+* If a person submits 100 translations, 20 are rejected, 20 are self-approved, the approval ratio will be 75% (60 over 80).
+
+The graph displays both the selected month’s ratio and the average for the previous 12 months.
```

Comment:
Reading the rest of the spec made me realize this chart is not supposed to update with the contribution chart.

I'd replace "selected" with "particular" or "specific" to avoid the confusion.